service/library/up.go

```diff
@@ -88,25 +88,51 @@ func Up(c Config) {
 		}
 	}
 
+	// Docker network(s) creation
```

Comment:
If this loop is specifically for network creation and not making connections should we drop support for the `Containers` property in the network array? It would simplify this significantly.